crates/rspack_core/src/normal_module_factory.rs

```diff
@@ -162,7 +162,7 @@ impl NormalModuleFactory {
       plugin_driver
         .normal_module_factory_resolve_for_scheme(ResourceData::new(
           request_without_match_resource.to_string(),
-          "".into(),
+          request_without_match_resource.to_string().into(),
```

Comment:
Same as webpack, when the scheme is data, resourcePath is the same as resource

Reply:
Are you sure? If I remember correctly, it should be undefined in webpack

Reply:
the resourcePath will be recalculated in js via the `resource` field, i have discard this change.